index.bs

```diff
@@ -1003,6 +1015,8 @@ An aggregatable contribution is a [=struct=] with the following items:
 :: A non-negative 128-bit integer.
 : <dfn>value</dfn>
 :: A non-negative 32-bit integer.
+: <dfn>filtering ID</dfn> (default: [=default filtering ID value=])
```

Comment:
Is the default value used? Similar for the default in other structs.

Reply:
I suppose that the default values are always set from the constants directly. As such, they can be removed. I removed them.